test/test_torch.py

```diff
@@ -6740,6 +6740,7 @@ def test_sobolengine_fast_forward(self, scramble: bool = False):
     def test_sobolengine_fast_forward_scrambled(self):
         self.test_sobolengine_fast_forward(scramble=True)
 
+    @skipIfTorchDynamo("np.float64 restored as float32 after graph break.")
```

Comment:
This is a known issue, we supported ```np.float64``` previously but someone changed it for some reasons.